Add T2T FAQ (#3419)
MicrosoftDocs/OfficeDocs-SharePoint: migration/mover-retirement-timeline.md

**Question:** Will individual or consumer migrations continue to work on Mover?</br>
**Answer:**   We'll continue to support individual, consumer, or student migrations from Mover for some time. Refer back to this page for updates on these migration scenarios. 

**Question:** Is tenant-to-tenant migration supported in Migration Manager?</br>

> **Answer:**  No. Microsoft will be offering a solution that will be available sometime in 2023. This will be a premium feature distinct from legacy Mover. In the interim, we recommend using a third-party solution for your Microsoft 365 tenant-to-tenant migration needs.

Reviewer: ```suggestion
**Answer:**  No. Microsoft is expecting to offer a solution sometime in 2023. The solution is expected to be a premium feature that's distinct from legacy Mover. In the interim, we recommend using a third-party solution for your Microsoft 365 tenant-to-tenant migration needs.
```

Reviewer: My understanding is that Microsoft Docs does not publish definite dates or features of future releases.

**Question:** Will individual or consumer migrations continue to work on Mover?</br>
**Answer:**   We'll continue to support individual, consumer, or student migrations from Mover for some time. Refer back to this page for updates on these migration scenarios. 

> **Question:** Is Tenant to Tenant (T2T) migration supported in Migration Manager?</br>

Reviewer 2: ```suggestion
**Question:** Is tenant-to-tenant migration supported in Migration Manager?</br>
```

> **Answer:**  No. Microsoft will be offering a solution that will be available sometime in 2023. This will be a premium feature distinct from legacy Mover. In the interim, we recommend using a third-party solution for your M365-to-M365 migration needs.

Reviewer 2: ```suggestion
**Answer:**  No. Microsoft will be offering a solution that will be available sometime in 2023. This will be a premium feature distinct from legacy Mover. In the interim, we recommend using a third-party solution for your Microsoft 365 tenant-to-tenant migration needs.
```
